src/app.js

```diff
@@ -316,7 +316,10 @@ function wsConnectionHandler(client, upgradeReq) {
             ampSessionId: meta.sessionId,
             ampUserId: meta.userId,
             ampDeviceId: meta.deviceId,
-            statsFormat: connectionInfo.statsFormat
+            statsFormat: connectionInfo.statsFormat,
+            isBreakoutRoom: meta.isBreakoutRoom,
+            roomId: meta.roomId,
```

Comment:
si roomId specific to only breakour rooms? if so maybe we can call it breakoutRoomId so we don't confuse it with the already existing "id" fields, wdyt?

Reply:
Yeah, I'll change it